fs/operations/dedupe.go

```diff
@@ -18,16 +18,23 @@ import (
 )
 
 // dedupeRename renames the objs slice to different names
-func dedupeRename(remote string, objs []fs.Object) {
-	f := objs[0].Fs()
-	doMove := f.Features().Move
+func dedupeRename(f fs.Fs, remote string, objs []fs.Object) {
+	info := objs[0].Fs()
```

Comment:
Now that you are passing in `f` you can remove this line and just use `f` in place of `info`